lib/rubocop/cop/layout/block_alignment.rb

```diff
@@ -22,23 +22,24 @@ module Layout
       #   # bad
       #
       #   foo.bar
-      #      .each do
-      #        baz
-      #          end
+      #     .each do
+      #       baz
+      #         end
       #
       #   # good
       #
-      #   variable = lambda do |i|
-      #     i
+      #   foo.bar
+      #     .each do
+      #       baz
       #   end
       #
       # @example EnforcedStyleAlignWith: start_of_block
       #   # bad
       #
       #   foo.bar
-      #      .each do
-      #        baz
-      #          end
```

Comment:
[memo] This indentation also changes at .each do with bad/good, which seemed confusing, so we are fixing that.